sdks/corda/src/main/kotlin/com/weaver/corda/sdk/AssetTransferSDK.kt

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.weaver.corda.sdk;
```

Comment:
Package name already has `sdk`, so may be rename the file to `AssetTransfer.kt` or `AssetTransferManager.kt`.

Reply:
As discussed, we have been using Manager all over (interop, sdk, application) which confusing. Hence, used the name SDK for better readability.